### PR TITLE
[FLOC 4360] Fix broken links in GCE and Kubernetes docs

### PR DESCRIPTION
**Pull request by davidoliverSP2**
Fixes: https://clusterhq.atlassian.net/browse/FLOC-4360

### CODE REVIEW

File: docs/installation/setup-gce.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ You'll need to setup at least two nodes.
 
 #. Create a new cloud server:
 
-   * Visit the `Google Cloud Console <https://console.cloud.google.com/>`_.
+   * Visit the `Google Cloud Console <https://console.cloud.google.com/start>`_.
```
Comment (zubron) [inline docs/installation/setup-gce.rst]: This is still resulting in a `404`. I've tried https://cloud.google.com/console locally which resulted in a successful link check build with the other change from below.

**Comment by zubron** [comment]
Thanks for making these changes so quickly @davidoliverSP2! Unfortunately the Google Cloud Console link is still giving a `404`. If the suggested link also works on Jenkins, then go ahead and merge 👍

File: docs/installation/setup-gce.rst (hunk 1 above)
Comment (sarum90) [inline docs/installation/setup-gce.rst]: Okay... so Google 404's if you aren't logged in, that's annoying. https://cloud.google.com/compute/docs/console might be an okay alternative.

Comment (sarum90) [inline docs/installation/setup-gce.rst]:
```
 ♦ ~$ curl --head https://console.cloud.google.com/
HTTP/1.1 404 Not Found
Content-Type: text/html; charset=utf-8
x-pan-config-state-hash: 1
x-pan-versionid: imagined-walker-201603282057-rc02.391748781147929096
X-Frame-Options: DENY
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Fri, 01 Jan 1990 00:00:00 GMT
Date: Tue, 05 Apr 2016 15:53:26 GMT
Server: Google Frontend
Alternate-Protocol: 443:quic
Alt-Svc: quic=":443"; ma=2592000; v="32,31,30,29,28,27,26,25"
Transfer-Encoding: chunked

 ♦ ~$ curl --head https://console.cloud.google.com/home
HTTP/1.1 404 Not Found
Content-Type: text/html; charset=utf-8
x-pan-config-state-hash: 1
x-pan-versionid: imagined-walker-201603282057-rc02.391748781147929096
X-Frame-Options: DENY
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Fri, 01 Jan 1990 00:00:00 GMT
Date: Tue, 05 Apr 2016 15:53:29 GMT
Server: Google Frontend
Alternate-Protocol: 443:quic
Alt-Svc: quic=":443"; ma=2592000; v="32,31,30,29,28,27,26,25"
Transfer-Encoding: chunked

 ♦ ~$ curl --head https://console.cloud.google.com/ho
 ♦ ~$ curl --head https://cloud.google.com/compute/docs/console
HTTP/1.1 200 OK
Strict-Transport-Security: max-age=31536000; includeSubdomains
X-Content-Type-Options: nosniff
Content-Language: en
Expires: Tue, 05 Apr 2016 16:53:49 GMT
Vary: Accept-Language
Last-Modified: Fri, 01 Apr 2016 19:12:05 GMT
X-XSS-Protection: 1; mode=block
Cache-Control: must_revalidate, public, max-age=3600
X-Frame-Options: SAMEORIGIN
Content-Type: text/html; charset=utf-8
Date: Tue, 05 Apr 2016 15:53:49 GMT
Server: Google Frontend
Alternate-Protocol: 443:quic
Alt-Svc: quic=":443"; ma=2592000; v="32,31,30,29,28,27,26,25"
Transfer-Encoding: chunked
```

Comment (sarum90) [inline docs/installation/setup-gce.rst]: Actually I've found that we have a list of exceptions. So long as https://console.cloud.google.com/ works for everyone here, I propose we just add that as an exception. I'm going to push a few commits on this branch, just so I can re-kick the tests.

**Comment by sarum90** [comment]
@wallrj, As far as I'm concerned this is ready to merge (sphinx, linkcheck, and lint all passed)

Would you mind double checking I didn't add a silly typo or something in my two commits, and then merge whenever.

**Comment by wallrj** [comment]
Looks good to me. The docs builder has finished  and passed so I'll merge.

**Pull request merged**